data/translations/bn/state-data.json

```diff
@@ -3,311 +3,407 @@
-        "confirm_registration_link": "https://olvr.hawaii.gov/"
+        "ip_deadline": "মঙ্গলবার, ৮ নভেম্বর, ২০২২",
+        "online_deadline": "মঙ্গলবার, ৮ নভেম্বর, ২০২২",
+        "mail_postmarked_deadline": "পোস্ট চিহ্ন করা আবশ্যক (সোমবার, ৩১ অক্টোবর, ২০২২)",
+        "confirm_registration_link": "https://olvr.hawaii.gov/",
+        "default_ip_deadline": "নির্বাচনের দিন পর্যন্ত এবং সহ উপলব্ধ",
+        "default_online_deadline": "নির্বাচনী দিনের ৩০ দিন আগে",
+        "default_mail_postmarked_deadline": "নির্বাচনের দিন ৮ দিন আগে পোস্টমার্ক করা আবশ্যক"
     },
     "id": {
         "hp_link": "https://idahovotes.gov/",
         "registration_link": "https://elections.sos.idaho.gov/ElectionLink/ElectionLink/ApplicationInstructions.aspx",
         "more_info_link": "https://idahovotes.gov/voting/",
-        "ip_deadline": "নির্বাচনের দিন পর্যন্ত এবং সহ উপলব্ধ",
-        "online_deadline": "নির্বাচনের ২৫ দিন আগে",
-        "mail_postmarked_deadline": "নির্বাচনের দিন ২৫ দিন আগে পোস্টমার্ক করা আবশ্যক",
-        "confirm_registration_link": "https://elections.sos.idaho.gov/ElectionLink/ElectionLink/VoterSearch.aspx"
+        "ip_deadline": "মঙ্গলবার, ৮ নভেম্বর, ২০২২",
+        "online_deadline": "শুক্রবার, ১৪ অক্টোবর, ২০২২",
+        "mail_postmarked_deadline": "পোস্ট চিহ্ন করা আবশ্যক (সোমবার, ৩১ অক্টোবর, ২০২২)",
+        "confirm_registration_link": "https://elections.sos.idaho.gov/ElectionLink/ElectionLink/VoterSearch.aspx",
+        "default_ip_deadline": "নির্বাচনের দিন পর্যন্ত এবং সহ উপলব্ধ",
+        "default_online_deadline": "নির্বাচনের ২৫ দিন আগে",
+        "default_mail_postmarked_deadline": "নির্বাচনের দিন ২৫ দিন আগে পোস্টমার্ক করা আবশ্যক"
     },
     "il": {
         "hp_link": "https://www.elections.il.gov/",
         "registration_link": "https://ova.elections.il.gov/",
         "more_info_link": "https://www.elections.il.gov/Default.aspx",
-        "ip_deadline": "নির্বাচনের দিন পর্যন্ত এবং সহ উপলব্ধ",
-        "online_deadline": "নির্বাচনের ১৬ দিন আগে",
-        "mail_received_deadline": "নির্বাচনের দিন ২৮ দিন আগে পেতে হবে",
-        "confirm_registration_link": "https://ova.elections.il.gov/RegistrationLookup.aspx"
+        "ip_deadline": "মঙ্গলবার, ৮ নভেম্বর, ২০২২",
+        "online_deadline": "মঙ্গলবার, ১৮ অক্টোবর, ২০২২",
+        "mail_postmarked_deadline": "পোস্ট চিহ্ন করা আবশ্যক (বৃহস্পতিবার, ৬ অক্টোবর, ২০২২)",
+        "confirm_registration_link": "https://ova.elections.il.gov/RegistrationLookup.aspx",
+        "default_ip_deadline": "নির্বাচনের দিন পর্যন্ত এবং সহ উপলব্ধ",
+        "default_online_deadline": "নির্বাচনের ১৬ দিন আগে",
+        "default_mail_postmarked_deadline": "নির্বাচনের দিন ২৮ দিন আগে পোস্টমার্ক করা আবশ্যক"
     },
     "in": {
         "hp_link": "https://www.in.gov/sos/elections/",
         "registration_link": "https://indianavoters.in.gov/",
         "more_info_link": "https://www.in.gov/sos/elections/2403.htm",
-        "ip_deadline": "নির্বাচনের দিন ২৯ দিন আগে",
```

Comment:
We need to keep the current `ip_deadline` as a `default_` value for fallback purposes.

Reply:
fixed